### PR TITLE
yoav & gil - Rotatet

### PR DESCRIPTION
**Pull request by asaf8888**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/swerve/chassis/SwerveChassis.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public enum Module {
 
 	public static final double TRANSLATION_TOLERANCE = 0.05;
 
-	public static final Rotation2d ROTATION_TOLERANCE = Rotation2d.fromDegrees(5);
+	public static final Rotation2d ROTATION_TOLERANCE = Rotation2d.fromDegrees(3);
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/subsystems/swerve/chassis/SwerveChassis.java:62]: Why not in swerve chassis constants?

File: src/main/java/edu/greenblitz/robotName/subsystems/swerve/chassis/SwerveChassis.java [hunk 2]
```diff
@@ -238,7 +239,7 @@ public void resetPoseByVision() {
 	 * returns chassis angle in radians
 	 */
 	public Rotation2d getGyroAngle() {
-		return Rotation2d.fromRadians(gyroInputs.yaw);
+		return Rotation2d.fromRadians(-gyroInputs.yaw);
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/subsystems/swerve/chassis/SwerveChassis.java:242]: Do the - in the pigeonGyro2 class at the update inputs

**Review by NoamRosenberg08 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 3]
```diff
@@ -22,113 +23,114 @@
...
+    private final SmartJoystick thirdJoystick;
+
+    private final SmartJoystick fourthJoystick;
+
+    private OI() {
+        mainJoystick = new SmartJoystick(RobotConstants.Joystick.MAIN);
+        secondJoystick = new SmartJoystick(RobotConstants.Joystick.SECOND);
+        thirdJoystick = new SmartJoystick(RobotConstants.Joystick.THIRD);
+        fourthJoystick = new SmartJoystick(RobotConstants.Joystick.FOURTH);
+    mainJoystick.A.whileTrue(new RotateToAngle(Rotation2d.fromDegrees(90)));
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/OI.java:47]: No?

**Review by yonnbro — APPROVED** [review]
Correct the one comment

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by yonnbro — APPROVED** [review]

**Pull request merged**